src/Aspire.Hosting.AppHost/Aspire.Hosting.AppHost.csproj

```
@@ -16,7 +16,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <None Include="**/*.props;**/*.targets" Pack="true" PackagePath="%(RecursiveDir)%(Filename)%(Extension)" />
+    <None Include="**/*.props;**/*.targets;*.json" Pack="true" PackagePath="%(RecursiveDir)%(Filename)%(Extension)" />
```

```suggestion
    <None Include="**/*.props;**/*.targets" Pack="true" PackagePath="%(RecursiveDir)%(Filename)%(Extension)" />
    <None Include="AspireAppHostConfiguration.json" Pack="true" />
```

Maybe we should just pack the one file we want, that way we don't accidently start including other .json files we don't intend to put in the nupkg.

---

sure, fixed it